isis/src/base/apps/hist/main.cpp

```diff
@@ -66,54 +66,61 @@ void IsisMain() {
   p.Progress()->CheckStatus();
   LineManager line(*icube);
 
-  for(int i = 1; i <= icube->lineCount(); i++) {
+  for (int i = 1; i <= icube->lineCount(); i++) {
     line.SetLine(i);
     icube->read(line);
     hist->AddData(line.DoubleBuffer(), line.size());
     p.Progress()->CheckStatus();
   }
 
-  if(!ui.IsInteractive() || ui.WasEntered("TO") ) {
```

Comment:
Why was the interactive check removed? It's unclear what this was supposed to do. It looks like it was suppose to write out to standard out, but I don't think it actually did that.